src/verifact_manager.py

```
@@ -10,17 +10,19 @@
 """
 
 import asyncio
+from re import A, I
```

Ruff (0.11.9)

`re.A` imported but unused

Remove unused import

(F401)